css.md

```diff
@@ -94,3 +95,10 @@ It's also a helpful pattern to prefix selectors (i.e. `.o-, .c-, .u-`) using the
 - `%` is also appropriate and can be used for column widths, etc.
 
 **[⬆  back to top](#table-of-contents)**
+
+## Stacking Order
+
+<a name="zindex"></a><a name="4.1"></a>
+[3.1](#zindex) Use variables or maps to manage z-index values. Variables or keys should be named semantically, such as "z-index-modal". Values should ideally be set `1, 2, 3, 4` etc. and shouldn't be set arbitrarily. Store all z-index variables or maps in a partial or single place.
```

Comment:
Suggestion: "... shouldn't be set arbitrarily. In other words, declarations like `z-index: 9999;` should never be used as a rule, similarly to avoiding `!important`. Store all z-index..."